.github/workflows/publish.yml

```diff
@@ -58,21 +56,35 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '16.13.0'
-          registry-url: "https://npm.pkg.github.com"
-          # Defaults to the user or organization that owns the workflow file
+          registry-url: "https://registry.npmjs.org"
           scope: "@zondax"
       - name: Install yarn
         run: npm install -g yarn
-      - name: Install modules
-        run: yarn install
       - name: Build package
-        run: yarn build
+        run: |
+          cd js
+          yarn install
+          yarn build
       - name: Get latest release version number
         id: get_version
         uses: battila7/get-version-action@v2
+      - name: Show version
+        run: echo ${{ steps.get_version.outputs.version }}
+      - name: Clean latest release version number
+        id: get_version_cleaned
+        uses: bhowell2/github-substring-action@v1
+        with:
+          output_name: version
+          value: ${{ steps.get_version.outputs.version }}
+          index_of_str: "npm_"
```

Comment:
this is old code..  from another project.. in that project to separate between releases.. we were using `npm_v` as a prefix.. 
That's the reason we have this substring step..  I think this whole thing about "cleaning" and using  `get_version_cleaned.outputs.version`  can be removed